client/app/components/tagging/tagging.directive.js

```diff
@@ -144,8 +144,8 @@
         return {id: tag.id,
                 category: {id: tag.category.id},
                 categorization: {
-                  display_name: tag.categorization.display_name,
-                }
+                  displayName: tag.categorization.display_name,
+                },
```

Comment:
Hi, this breaks tagging when initiated from the blueprint details because it is still using display_name.  I'm looking at the best fix would be...stay tuned :-)